Jenkinsfile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env groovy
-common {
-  slackChannel = '#clients-eng'
-  upstreamProjects = 'confluentinc/rest-utils'
+
+docker_oraclejdk8 {
+  dockerPullDeps = ['confluentinc/cp-base']
+  dockerPush = true
+  dockerRegistry = '368821881613.dkr.ecr.us-west-2.amazonaws.com/'
+  dockerRepos = ['confluentinc/cp-schema-registry']
+  dockerUpstreamTag = '4.0.x-latest'
```

Comment:
is version bumping setup to take this new setting into account? also, is it possible to set these just based on the branch/tag that's being built instead of having to have them here as well?

Reply:
I'll see what can be done here.